**Ready for Review** |Visualization of RNA-Seq results with Volcano Plot |updated the workflow, tutorial, tools, and text. (#5949)
galaxyproject/training-material: topics/transcriptomics/tutorials/rna-seq-viz-with-volcanoplot/tutorial.md

>
>    {% snippet faqs/galaxy/histories_rename.md %}
>

> > 2. Import the result table of differential expression analysis as well as a list of genes to annotate in the volcano plot in the end of this tutorial.

Reviewer: ```suggestion
> 2. Import the result table of differential gene expression analysis, as well as a list of genes that will be annotated in the volcano plot later.
```
